shim.js

```diff
@@ -21,7 +20,7 @@ module.exports = function shimName() {
 			configurable: true,
 			enumerable: false,
 			get: function () {
-				var name = fnCall(polyfill, this);
+				var name = bind.call(polyfill, this).call(this);
```

Comment:
@ljharb It doesn't seem like this is robust against `delete Function.prototype.call` either, do you have a better idea? 🔨